validationTemplate/src/ROMinput/trainROM.xml

```diff
@@ -0,0 +1,153 @@
+<Simulation verbosity="debug">
+    <TestInfo>
+        <author>mgarrou</author>
+        <created>2021-11-29</created>
+        <description>
+            Run Dymola model and train a ROM from the resulting data (time-dependent)
+        </description>
+    </TestInfo>
+
+    <RunInfo>
+        <WorkingDir>trainROMWorkDir</WorkingDir>
+        <Sequence>runDymolaTEDS, trainROM, printAccuracy, dumpPickledROM</Sequence>
+        <batchSize>1</batchSize>
+    </RunInfo>
+
+    <Files>
+        <Input name="dsin.txt" type="DymolaInitialisation">C:\Users\garrm\Documents\SimTEDS\initial_exp_data\update_pipe_length\dsin.txt</Input>
```

Comment:
Ideally this would be a relative path based on some spot we would place the files.